web/components/nav/group-sidebar.tsx

```diff
@@ -0,0 +1,87 @@
+import { HomeIcon, BookOpenIcon } from '@heroicons/react/outline'
+import clsx from 'clsx'
+import { useUser } from 'web/hooks/use-user'
+import { ManifoldLogo } from './manifold-logo'
+import { ProfileSummary } from './profile-menu'
+import React from 'react'
+import TrophyIcon from 'web/lib/icons/trophy-icon'
+import { SignInButton } from '../sign-in-button'
+import CornerDownRightIcon from 'web/lib/icons/corner-down-right-icon'
+import NotificationsIcon from '../notifications-icon'
+import { SidebarItem } from './sidebar'
+import { buildArray } from 'common/util/array'
+import { User } from 'common/user'
+
+const groupNavigation = [
+  { name: 'About', key: 'about', icon: BookOpenIcon },
+  { name: 'Markets', key: 'markets', icon: HomeIcon },
+  { name: 'Leaderboard', key: 'leaderboards', icon: TrophyIcon },
+]
+
+const generalNavigation = (user?: User | null) =>
+  buildArray(
+    user && {
+      name: 'Notifications',
+      href: `/notifications`,
+      key: 'notifications',
+      icon: NotificationsIcon,
+    }
+  )
+
+export default function GroupSidebar(props: {
```

Comment:
We don't typically use default exports, except for pages. It's good to be consistent so you always know how to import anything.

Two reasons for this convention: 1. can easily export more things from a file without updating all references 2. Renaming an export will automatically rename it everywhere it is referenced, which is not true for default exports.